### PR TITLE
Princing table changes

### PR DESCRIPTION
**Pull request by ThiagoDallacqua**
This PR intends to fulfill the following Jira tickets: [MMS-18](https://murcul.atlassian.net/browse/MMS-18), [MMS-19](https://murcul.atlassian.net/browse/MMS-19) and [MMS-20](https://murcul.atlassian.net/browse/MMS-20)

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1922,7 +1922,6 @@
       "requires": {
         "anymatch": "1.3.2",
         "async-each": "1.0.1",
-        "fsevents": "1.1.3",
```
Comment (ziahamza) [inline package-lock.json]: @ThiagoDallacqua how come this file changed?

Comment (ThiagoDallacqua) [reply package-lock.json]: `npm install` I guess...

Comment (ThiagoDallacqua) [reply package-lock.json]: I've created the branch based on `master`, so perhaps it is updated and when I ran `npm install` after cloning it updated

**Comment by ThiagoDallacqua** [comment]
@ziahamza fixed it

**Pull request merged**